templates/cinder/bin/init.sh

```diff
@@ -26,51 +26,59 @@ export DBPASSWORD=${DatabasePassword:?"Please specify a DatabasePassword variabl
 export PASSWORD=${CinderPassword:?"Please specify a CinderPassword variable."}
 export TRANSPORTURL=${TransportURL:-""}
 
-export CUSTOMCONF=${CustomConf:-""}
-
+DEFAULT_DIR=/var/lib/config-data/default
+CUSTOM_DIR=/var/lib/config-data/custom
+MERGED_DIR=/var/lib/config-data/merged
 SVC_CFG=/etc/cinder/cinder.conf
-SVC_CFG_MERGED=/var/lib/config-data/merged/cinder.conf
-
-# expect that the common.sh is in the same dir as the calling script
-SCRIPTPATH="$( cd "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P )"
-. ${SCRIPTPATH}/common.sh --source-only
+SVC_CFG_MERGED=${MERGED_DIR}/cinder.conf
+SVC_CFG_MERGED_DIR=${MERGED_DIR}/cinder.conf.d
 
-# Copy default service config from container image as base
-cp -a ${SVC_CFG} ${SVC_CFG_MERGED}
+mkdir -p ${SVC_CFG_MERGED_DIR}
 
-# Merge all templates from config-data defaults first, then custom
-# NOTE: custom.conf files (for both the umbrella Cinder CR in config-data/defaults
-#       and each custom.conf for each sub-service in config-data/custom) still need
-#       to be handled separately below because the "merge_config_dir" function will
-#       not merge custom.conf into cinder.conf (because the files obviously have
-#       different names)
-for dir in /var/lib/config-data/default /var/lib/config-data/custom; do
-    merge_config_dir ${dir}
-done
+cp ${DEFAULT_DIR}/* ${MERGED_DIR}
 
-# TODO: a cleaner way to handle this?
-# Merge custom.conf with cinder.conf, since the Kolla config doesn't seem
-# to allow us to customize the cinder command (it calls httpd instead).
-# Can we just put custom.conf in something like /etc/cinder/cinder.conf.d/custom.conf
-# and have it automatically detected, or would we have to somehow change the call
-# to the cinder binary to tell it to use that custom conf dir?
-echo merging /var/lib/config-data/default/custom.conf into ${SVC_CFG_MERGED}
-crudini --merge ${SVC_CFG_MERGED} < /var/lib/config-data/default/custom.conf
+# Save the default service config from container image as cinder.conf.sample,
+# and direct create a small cinder.conf file that directs people to files
```

Comment:
That should be "and create" (not "and direct create"). I can fix this if other updates are necessary, or it could be treated as an awkwardly worded nit.